tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -146,7 +146,44 @@ jobs:
    
     workingDirectory: '$(Build.BinariesDirectory)\$(BuildConfig)\$(BuildConfig)'
     displayName: 'Run tests'
- 
+
+  - task: PythonScript@0
+    displayName: 'Regenerate cmake config with STATIC_ANALYSIS=ON '
+    condition: and(succeeded(), eq(variables['BuildConfig'], 'Debug'))
+    inputs:
+      scriptPath: '$(Build.SourcesDirectory)\tools\ci_build\build.py'
+      arguments: '--config Debug --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --update --cmake_generator "Visual Studio 16 2019" --build_wheel --build_shared_lib --enable_onnx_tests --cmake_extra_defines onnxruntime_ENABLE_STATIC_ANALYSIS=ON'
```

Comment:
>--enable_onnx_tests [](start = 197, length = 19)

do we need stuff like enabling the onnx tests if we're doing static analysis, or can we just keep it to the core code (could maybe turn off building of all the unit tests to make it faster as well)

Reply:
The "enable_onnx_tests" build flags doesn't control if the unit tests would be built.  onnxruntime_BUILD_UNIT_TESTS  does that.  I think it's OK to keep it ON, it won't cost much time. Either works.